IdentityCore/src/util/MSIDJWTHelper.m

```diff
@@ -79,9 +80,7 @@ + (NSString *)decryptJWT:(NSData *)jwtData
 + (NSData *)sign:(SecKeyRef)privateKey
             data:(NSData *)plainData
 {
-
-    NSData *hashData = [plainData msidSHA256];
-    return [hashData msidSignHashWithPrivateKey:privateKey];
+    return [[MSIDKeyOperationUtil sharedInstance] getSignatureForDataWithKey:plainData privateKey:privateKey context:nil error:nil];
```

Comment:
It's unclear from this line what the new signature method is. Maybe add a comment indicating it's RSA+SHA256?

Reply:
The util will sign using the appropriate algorithm according to the key type of the private key. Will use RS256 if the privateKey is a RSA key and ES256 if it is a ECC key.